lib/std/os/bits/linux.zig

```diff
@@ -770,6 +770,9 @@ pub fn S_ISSOCK(m: u32) bool {
     return m & S_IFMT == S_IFSOCK;
 }
 
+pub const UTIME_NOW = 0x3fffffff;
+pub const UTIME_OMIT = 0x3ffffffe;
```

Comment:
Should these be expressed as `(@as(u32, 1) << 30) - 2`?

Reply:
I saw that libc defines them using left shifts, while musl defines the values directly. But using `@as(u32, ...)` would mean they need to be re-casted to be used in the context of a `timespec.tv_nsec` field (whose definition varies across architectures), I think? Other than that I have no preference.